Pipfile

```diff
@@ -0,0 +1,19 @@
+[[source]]
```

Comment:
I am not apposed to using a virtual env for Python that makes life easier but we do some checks here https://github.com/forem/selfhost/blob/citizen428/setup-script/roles/preflight/tasks/main.yml#L21-L43

That reads the `requirements.txt` file and it ensures that we have needed packages installed when Ansible runs. Is there a good way we can read the `Pipfile` instead and nuke the `requirements.txt` so we are not maintaining the reqs in two spots?

Reply:
Ah, I should have mentioned this, the `Pipfile` as automatically created by running `pipenv install`.

> If you only have a `requirements.txt` file available when running `pipenv install`, `pipenv` will automatically import the contents of this file and create a `Pipfile` for you.

We could completely remove `requirements.txt` at this point. We'd then have to change the preflight task to take the output of `pipenv run pip freeze` instead of reading a requirements file which probably isn't too hard.

Reply:
I have only really used a requirements.txt with Ansible, so I am OK with leaving things as is and we can refactor things in Ansible to read the output of `pipenv run pip freeze` later. ™️ 

If we make adjustments to the `requirements.txt` file how do we update the Pipfile? Manually or is there a magic pipenv command to sync them?

Reply:
I just saw your commits to remove the `requirements.txt` file. Looks great. I think my only non blocking gripe here is it now forces everyone to use the pipenv to run selfhost vs system ansible/python whether they use the script or not.

Reply:
tl;dr: I'll remove `pipenv`.

> it now forces everyone to use the pipenv to run selfhost vs system ansible/python whether they use the script or not

I guess I'm approaching this from a developer perspective where it's relatively uncommon to use a system installed Python and no virtual envs. But I understand that this may be different for system folks. 

The reason I prefer `pipenv` or similar is not even that they have lots of extra features like `pipenv check` ("Checks for PyUp Safety security vulnerabilities and against PEP 508 markers provided in Pipfile") but that they keep each project's dependencies isolated so you don't have to worry about version conflicts. 

But I see your point. Python 3.3+ has native support for virtual environments via the `venv` lib in the standard library, I guess people who care enough about having their system Python untouched know or can figure out how to use that. I'm personally not the biggest fan of this because it requires you to use Bash or ZSH as a shell and doesn't work with Fish out of the box, but again, people who made these choices should know what to do (e.g. `exec bash -c "source env/bin/activate; exec fish"`).

Reply:
`pipenv` is gone now. I can add a note to the README about using `venv`.